src/client/dss.py

```diff
@@ -42,44 +49,74 @@ def initiate(printer, action, target=None, file=None):
         common.setup_client_from_env_vars()
         common._ensure_global_vars_populated()
         if common.global_vars['dss_url'] is None:
-            print "Global variable dss_url not set!"
+            print "Global variable DSS_URL not set!"
```

Comment:
This is okay... but the actual global var is dss_url and is present in common.py. DSS_URL is the env var.